cdk/lib/prism.ts

```diff
@@ -72,7 +75,10 @@ export class PrismStack extends GuStack {
 
     const asg = new GuAutoScalingGroup(this, "AutoscalingGroup", {
       ...PrismStack.app,
-      existingLogicalId: "AutoscalingGroup",
+      existingLogicalId: {
+        logicalId: "AutoscalingGroup",
+        reason: "We override this to ensure that we do not replace the existing resource",
```

Comment:
Worth being a little more descriptive? e.g. is this stateful and replacing it will result in downtime?

Reply:
I had to pause whilst copy & pasting this one... I'm not actually completely sure if replacing an ASG is a problem? I guess it depends whether AWS waits for the new ASG's boxes to come into service before deleting the old one...